guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -12,138 +12,250 @@
    ]
```

Comment:
<p>IS this an alternative to using the geo-enrichment source from AGOL? Is so, then we should not use <code>arcgis_pro</code> and <code>arcgis_agol</code> respectively in the rendering and data access statements (in the later cells), instead we might just use a single variable indicating that these two are alternative to each other.</p><p>Also, is Arcgis Pro with BA extension a pre-requisite condition? If so, we might need to state in an <code>Important Note</code> or <code>Pre-requisite</code> highlighting statement??   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>